src/desktop/apps/armory_week/client/initializer.tsx

```diff
@@ -3,6 +3,7 @@ import ReactDOM from "react-dom"
 import { BannerPopUp } from "desktop/components/fair_week_marketing/BannerPopUp"
 
 export const init = () => {
+  // @ts-ignore
```

Comment:
Can we [update the typings file](https://github.com/damassi/force/blob/a24de41815c6fdf5770aa15c8cfd36624313fed3/src/typings/global.d.ts#L1) with `__BOOTSTRAP__: any`?